Base/etc/services

```diff
@@ -392,7 +392,7 @@ afs3-vlserver	7003/tcp			# volume location database
 afs3-vlserver	7003/udp
 afs3-kaserver	7004/tcp			# AFS/Kerberos authentication
 afs3-kaserver	7004/udp
-afs3-volser	7005/tcp			# volume managment server
```

Comment:
I'd rather see this fixed in the IANA registry.

Reply:
True - could you point me to a repo or mailing list? Not exactly sure where this is coming from.

Reply:
Oops, it says at the top of the file. https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml

Reply:
I have no idea about their contribution processes. Must be extremely bureaucratic though.